e2e-tests/development-runtime/gatsby-config.js

```diff
@@ -12,6 +12,16 @@ module.exports = {
               variants: [`300`, `400`, `500`],
             },
           ],
+          selfHosted: [
+            {
+              family: `Open Sans`,
+              urls: {
+                woff2: `/OpenSans400.woff2`,
```

Comment:
We assume the fonts are in static folder and copied to the app root.
Here I am not sure if it would be better to specify the font file source.

```javascript
src: {
                woff2: path.join('static', `OpenSans400.woff2'),
}
```

What do you think?